package.json

```diff
@@ -39,19 +44,23 @@
     "uuid": "^8.1.0"
   },
   "scripts": {
-    "start": "REACT_APP_SOSS_SERVER=wss://localhost:50001 REACT_APP_SOSS_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjoicm9taS1kYXNoYm9hcmQiLCJpYXQiOjE1ODMyODYyMTV9.x9aNjcLujQPHchWEsbrRbvctmnGQtEzw-81X0aPIE-Y REACT_APP_TRAJECTORY_SERVER=ws://localhost:8006 react-scripts start",
+    "start": ". e2e/env.sh && run-p -r start:auth start:rmf start:react",
+    "start:react": "react-scripts start",
+    "start:react:prod": "serve -c ../e2e/serve.json build",
```

Comment:
> Just checking, should we build first? npm run build && ... or we will let the developer build manually?

yea, I thought splitting it up would allow for faster debug cycle, but that actually caused me some problems before as I forgot to rebuild it. I guess with `test:e2e:dev` there is very little reason to skip rebuilding, and if it's really needed it shouldn't be hard to temporary modify the script.

---

Is the auth server running? Currently it just get a white screen when the auth server is not contactable, I can add a error page if that happens.

Reply:
The keycloak adapter doesn't throw any error when the server is not online, it just get stucked there indefinitely. I can put a timeout but that could be misleading as the browser shows the page as having completely loaded (not spinning icon). Under normal circumstances the auth server should always be contactable, do you think the current behavior is ok?

Reply:
Yeah, makes sense, I think it's ok.